src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java

```diff
@@ -197,6 +218,39 @@ private static String getMypyConfigFile(Project project, String mypyConfigFilePa
         return mypyConfigFilePath;
     }
 
+    public static String detectVenvMypyPath(Project project) throws IOException {
+        VirtualFile interpreterFile = getInterpreterFile(project);
+        String interpreterPath = interpreterFile != null ? interpreterFile.getPath() : null;
+        if (interpreterPath == null) {
+            return null;
+        }
+        String venvPath =
+                Paths.get(PathUtil.getParentPath(PathUtil.getParentPath(interpreterPath)), "Scripts").toString();
+        return findFileByName("mypy", venvPath, "glob:**/*.exe");
```

Comment:
If I'm not wrong, this detection will work only on Windows and the plugin support also Linux and Mac OSs.

Could you please adapt this method to support also the other OSs?

For Linux and Mac the mypy binary is located here: `$venvPath/bin/mypy` (keep in mind that there is no `.exe` at the end of the binary file on Linux and Mac).